core/commands/swarm.go

```diff
@@ -50,7 +52,14 @@ var swarmPeersCmd = &cmds.Command{
 'ipfs swarm peers' lists the set of peers this node is connected to.
 `,
 	},
+	Options: []cmds.Option{
+		cmds.BoolOption("verbose", "v",
+			`Also display latency along with peer information in the following form:
+        <peer address> <latency>
```

Comment:
You are using raw string here. Meaning that now this string includes two new lines and also indentation. Please change it to one line.

If you want to split line you can do it like that
```go
"line1" +
"still line1"
```